src/components/loginForm/LoginForm.tsx

```diff
@@ -12,12 +14,15 @@ const LoginForm: React.FunctionComponent = () => {
   const [email, setEmail] = React.useState<string>("");
   const [password, setPassword] = React.useState<string>("");
   const [isLoading, setLoading] = React.useState<boolean>(false);
+  const [isError, setError] = React.useState<boolean>(false);
```

Comment:
I think you should use `loginState` enum with "ERROR", "LOADING", "SUCCESS" statuses instead of flags.